resources/netCore/GetProjectProperties.targets

```diff
@@ -1,11 +1,16 @@
 <Project>
-    <!-- All the relevant info is in root-level PropertyGroups, so there are no dependent targets to make this work -->
-    <Target Name="GetProjectProperties">
+    <PropertyGroup>
+        <GetProjectPropertiesDependsOn Condition=" '$(SDKContainerSupportEnabled)' == 'true' ">$(GetProjectPropertiesDependsOn);ComputeContainerConfig;</GetProjectPropertiesDependsOn>

```

Comment:
This file still works when SDK container support is not enabled, right?

Reply:
yeah,`$(OutputPath)$(AssemblyName).dll` will remain unchanged